string actionType
string objectTitle
string objectType
string objectCategory
string roomSubject
string furnitureSubject
string roomTarget
string furnitureTarget
---
